packages/@aws-cdk/aws-cognito-identitypool-alpha/README.md

```diff
@@ -179,19 +180,19 @@ new IdentityPool(this, 'myidentitypool', {
   identityPoolName: 'myidentitypool',
   authenticationProviders: {
     amazon: {
-      appId: 'amzn1.application.12312k3j234j13rjiwuenf',
+      clientId: 'amzn1.application.12312k3j234j13rjiwuenf',
```

Comment:
this seems to be a breaking change though, can we mention in the PR.

Reply:
No longer needed, as I removed the breaking change